schedule/yast/autoyast_non_secure_boot.yaml

```diff
@@ -0,0 +1,23 @@
+name:           autoyast_non_secure_boot
+description:    >
+  Test AutoYaST installation with non-secure boot option. 
+  Covered cases:
+     - Verify that installed system in uefi mode, doesn't have secure boot.  
+vars:
+  AUTOYAST: autoyast_sle15/autoyast_non_secure_boot.xml
+  AUTOYAST_PREPARE_PROFILE: 1
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/wicked
+  - autoyast/repos
+  - autoyast/clone
```

Comment:
i dont think you need this module in this job suite

Reply:
I don't need lines [17..20]  but It's looks like most autoyast installation tests are using similar schedule, so I thought it was extra checks.

Reply:
autoyast/login : checks if you are in the login screen
autoyast/wicked : enable wicked and gather some info
autoyast/repos : I am not sure about that. you can remove and check
autoyast/clone : performs a clone_system that you dont need in this scenario. i might make sense if you want to verify the generated profile but this is another story

Reply:
You have a point, therefore I added cloned profile verification module.

Reply:
@jknphy WDYT about the schedule? The test's requirements could be fulfilled by a schedule like following : https://openqa.suse.de/tests/3975140

Reply:
not really, we are using those extra checks, for example we always use `autoyast/clone`as there are plenty of bugs related with empty fields or wrong fields in the result xml. I spot something important here that we need to fix, that you didn't add `test_data` for `verify_cloned_profile` https://openqa.suse.de/tests/3980328#step/verify_cloned_profile/6 so we are not checking anything (we should add some check here, doesn't need to be in this PR to check for empty data provided and fail). You can check other profiles to see how is done this xpath validation with that module.

The rest of the module looks optional for me, login, wicked, logs, repos sometimes it is not because it is needed if you don't have bzip2, so it depends on the module you install in your ay profile. it is up to you this part.

Reply:
I have added the test_data for verification of cloned profile. About schedule, I think I should keep the extra modules as we don't have any uefi autoyast tests and I think they could be useful.

Reply:
I noticed you added verification for the partitioning, in fact for this test is not really required, due to all those checks are just default values, what is really interesting is to verify 'secure_boot' xml element. It would suffice with:
```
profile:
    bootloader:
      global:
          secure_boot: false
```
You could still keep current validations due probably we are not checking strictly default checks in any other autoyast scenario (perhaps some mix between default and specific) and add just that block above.

Reply:
Done